lib/Requestable.js

```diff
@@ -136,19 +137,19 @@ class Requestable {
     * A function that receives the result of the API request.
     * @callback Requestable.callback
     * @param {Requestable.Error} error - the error returned by the API or `null`
-    * @param {(Object|true)} result - the data returned by the API or `true` if the API returns `204 No Content`
-    * @param {Object} request - the raw {@linkcode https://github.com/mzabriskie/axios#response-schema Response}
+    * @param {axios.Response | boolean} result - the data returned by the API or `true` if the API returns `204 No Content`
+    * @param {axios.RequestOptions} request - the raw {@linkcode https://github.com/mzabriskie/axios#response-schema Response}
```

Comment:
I couldn't find `axios.Response` and `axios.RequestOptions` in axios/index.d.ts.
Also the callback signature doesn't seem to match how it is used [here](https://github.com/github-tools/github/blob/f075749ed4234c7d76f122879523fbca2eb01f86/lib/Requestable.js#L189):
```js
cb(null, response.data, response);
```
Shouldn't it be:
```ts
/**
 * A function that receives the result of the API request.
 * @callback Requestable.callback
 * @param {Requestable.Error} error - the error returned by the API or `null`
 * @param {any | boolean} result - the data returned by the API or `true` if the API returns `204 No Content`
 * @param {axios.AxiosResponse} response - the raw {@linkcode https://github.com/mzabriskie/axios#response-schema Response}
 */
type callback = (
  error: Requestable.Error,
  result: (any | boolean), // https://github.com/mzabriskie/axios/blob/4976816808c4e81acad2393c429832afeaf9664d/index.d.ts#L48
  response: axios.AxiosResponse // Not request but response
) => void;
```